docs/docs/agents/prebuilt.md

```diff
@@ -1,7 +1,44 @@
+[//]: # (This file is automatically generated using a script in docs/_scripts. Do not edit this file directly!)
```

Comment:
This gets generated during build, right? So I think we don't need to check it in